src/pages/events/configure-commerce.md

```diff
@@ -197,11 +197,11 @@ Commerce provides two sources for events: observers and plugins. You must specif
 1. If you don't have a module ready for integration with Adobe I/O Events, or you don't know exactly which events to register at this point, use the `events:subscribe` command to subscribe to some sample events, as shown in the following example commands:
 
    ```bash
-   bin/magento events:subscribe observer.catalog_product_save_after
+   bin/magento events:subscribe observer.catalog_product_save_after --fields sku --fields stock_data.qty
    ```
 
    ```bash
-   bin/magento events:subscribe observer.customer_login
+   bin/magento events:subscribe plugin.magento.sales.api.invoice_item_repository.save --fields parent_id --fields base_price --fields tax_amount --fields base_row_total --fields discount amount --fields qty
```

Comment:
It's better to add examples with `observer` plugin types. As you can't subscribe to the `plugin` event types if it wasn't in `config.php` before.

Reply:
```
bin/magento events:subscribe observer.customer_login --fields=firstname --fields=lastname --fields=entity_id
```